app/javascript/quantity_input.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
+  $(document).on("click", ".plus", function() {
+    var qtyScope = $(this).closest(".quantity-scope");
+    var input = qtyScope.find(".count");
+    var currentValue = parseInt(input.val());
+
+    input.val(currentValue + 1);
+    if (input.hasClass("count-update")) {
+      input.trigger("change");
+    }
+  });
+
+  $(document).on("click", ".minus", function() {
```

Comment:
```suggestion
  $(document).on("click", ".js-minus", function() {
```
nên tách biệt class dùng để css và class để dùng js ra nha em. để sau này maintain nó dể hơn vd như nhìn tiền tố js- là mình sẽ biết đc class này dùng chạy js,  lúc sửa or xóa nó k ảnh hưởng đến css

Reply:
Dạ oke a